### ISSUE TITLE
Collection-level content warning scope note should inherit down

### ISSUE THREAD

**Issue by faithc** [pulibrary/pulfalight#1237]
Like other collection-level notes, scopecontent notes with the label "content warning" should inherit down unless superseded by a lower-level note. See for example: https://findingaids.princeton.edu/catalog/TC040 

https://findingaids.princeton.edu/catalog/TC040_c00002 should have the content warning from https://findingaids.princeton.edu/catalog/TC040_c00001

## Sudden Priority Justification

Without this, our researchers will not be appropriately warned about potentially harmful content.

**Comment by tpendragon** [comment]
We want to treat scopecontent notes with the Content Warning header as special for the purpose of inheritance. Content warnings inherit down always.

**Comment by tpendragon** [comment]
https://github.com/pulibrary/pulfalight/tree/1237-content-warning-scope makes scopecontent inherit down (and fixes inheritance for all the notes we defined.) I'm blocked on the content warning special behavior portion of this ticket until I have an example where a parent has a content warning a component has a scopecontent but no content warning. @faithc @ccleeton ?

**Comment by faithc** [comment]
> We want to treat scopecontent notes with the Content Warning header as special for the purpose of inheritance. Content warnings inherit down always.
> 
> Lower levels can have their own content warnings, but can't remove a parent's content warning.

I'm not sure this is true; what you mean by "Lower levels can have their own content warnings, but can't remove a parent's content warning." A warning at a lower level should supersede a parent warning. For example, there could be a collection-level warning noting the existence of harmful content and a note at a lower level with explicit information about particular materials, if that makes sense. For example, the collection-level warning for TC040 is superseded by the note at [TC040_c00001](https://findingaids.princeton.edu/catalog/TC040_c00001).

**Comment by tpendragon** [comment]
@faithc What we had talked about was if a TC040 had a "Content Warning", and TC040_c00001 had a "Scope & Contents" note, then TC040_c00001 should have a "Scope & Contents" as well as a "Content Warning", and there's no way to make it so it doesn't.

I'm also hearing we can't find an example where the parent has a content warning and the child has a scope & content w/ no content warning, though.

**Comment by faithc** [comment]
Ok, right, yes. 

And I can't think of an example off the top of my head where this is true.

**Comment by tpendragon** [comment]
https://findingaids.princeton.edu/catalog/TC040_c01829?onlineToggle=false should have a content warning and does not.

**Comment by tpendragon** [comment]
I think this indexed, but did not display - maybe it only displays the `_combined` field if there's something in the `scopecontent` field.

**Comment by faithc** [comment]
The collection-level content warning that was applied to WC064, https://findingaids.princeton.edu/catalog/WC064, is not inheriting down.

Also, the collection-level scope and contents note is no longer displaying.
